javascript/javascript3/week2/currency-calculator.js

```diff
@@ -0,0 +1,84 @@
+const BASE_URL = "https://v6.exchangerate-api.com";
+const MY_KEY = "a7b1d475b4a2c818963b7f85";
+
+const currencies1 = {
```

Comment:
The task says: `Make a request to the API and store the Exchange rates as well as a list of currencies for the dropdowns.` Therefore both exchange rates and currencies should come from the API.